src/Files.App/Helpers/NaturalStringComparer.cs

```diff
@@ -8,7 +8,7 @@ public sealed class NaturalStringComparer
 	{
 		public static IComparer<object> GetForProcessor()
 		{
-			return new NaturalComparer(StringComparison.CurrentCulture);
+			return new NaturalComparer(StringComparison.CurrentCultureIgnoreCase);
```

Comment:
Btw this is instantiated every time `SortingHelper.OrderFileList` is called. We prob should create a static field for this (if null, for the first time, create an instance).

Reply:
Can you please start an issue so we can track that?